pkg/config/config_template.yaml

```diff
@@ -430,429 +787,405 @@ api_key:
+
+## @param kubelet_client_ca - string - optional
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #
-# We can extract pod labels and annotations as metric tags. If you prefix your
-# tag name with +, it will only be added to high cardinality metrics
+#  kubelet_client_ca: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+
+## @param kubelet_auth_token_path - string - optional
+## If authentication is needed, the Agent uses the pod's serviceaccount's
+## credentials. If you want to use a different account, or are running the Agent
+## on the host, set a custom token file path here.
 #
-# kubernetes_pod_labels_as_tags:
-#   app:               kube_app
-#   pod-template-hash: +kube_pod-template-hash
+#  kubelet_auth_token_path: <TOKEN_FILE_PATH>
+
+## @param kubelet_client_crt - string - optional
+## Set a custom Client CRT file path.
 #
-# kubernetes_pod_annotations_as_tags:
-#   app:               kube_app
-#   pod-template-hash: +kube_pod-template-hash
+#  kubelet_client_crt: <CRT_FILE_PATH>
+
+## @param kubelet_client_key - string - optional
+## Set a custom Client key file path.
 #
-{{ end -}}
-{{- if .ECS }}
-# ECS integration
+#  kubelet_client_key: <CLIENT_KEY_FILE_PATH>
+
+## @param kubelet_wait_on_missing_container - integer - optional - default: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## On some kubelet versions, containers can take up to a second to
+## register in the podlist. This option allows to wait for up to a given
+## number of seconds (in 250ms chunks) when a container does not exist in the podlist.
 #
-# The ECS agent container should be autodetected when running with the
-# default (ecs-agent) name. Else, you can change the container name the
-# agent will look for, or force a fixed url:
-# ecs_agent_container_name: ecs-agent
-# ecs_agent_url: http://localhost:51678
+#  kubelet_wait_on_missing_container: XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+## @param kubelet_cache_pods_duration - integer - optional - default: 5
+## Polling frequency in seconds of the Agent to the kubelet "/pods" endpoint.
 #
-# Fargate clusters use other endpoints and are not affected by these options.
+#  kubelet_cache_pods_duration: 5
+
+## @param kubernetes_pod_expiration_duration - integer - optional - default: 900
+## Set the time in second after which the Agent ignores the pods that have exited.
+## Set the duration to 0 to disable this filtering.
 #
+#  kubernetes_pod_expiration_duration: 900
+
 {{ end -}}
-{{- if .CRI }}
-# CRI integration
-#
-# To activate the CRI check you'll need to indicate the path of the
-# CRI runtime you're using (and mount it in the container if needed)
-# cri_socket_path: /var/run/containerd/containerd.sock
+{{- if .KubeApiServer }}
+
+####################################################
+## Kubernetes apiserver integration Configuration ##
+####################################################
+
+## @param kubernetes_kubeconfig_path - string - optional
+## When running in a pod, the Agent automatically uses the pod's serviceaccount
```

Comment:
```suggestion
## When running in a pod, the Agent automatically uses the pod's service account
```
Not sure about this one, since it's occurred more than once, but I've never seen this as one word before